data/tabular/perovskite_db/meta.yaml

```diff
@@ -0,0 +1,155 @@
+    - id: iupac_formulas 
+      type: Other
+      description: IUPAC chemical formula
+    - id: descriptive_formulas
+      type: Other
+      description: descriptive chemical formula
+    - id: device_stack_string
+      type: Other
+      description: device stack
+license: CC BY 4.0
+links:
+    - url: http://www.perovskitedatabase.com
+      description: original data source
+num_points: 29601
+bibtex:
+    - |-
+      @article{Jacobsson_2021,
+        doi = {10.1038/s41560-021-00941-3},
+        url = {https://doi.org/10.1038%2Fs41560-021-00941-3},
+        year = 2021,
+        month = {dec},
+        publisher = {Springer Science and Business Media {LLC}},
+        volume = {7},
+        number = {1},
+        pages = {107--115},
+        author = {T. Jesper Jacobsson and Adam Hultqvist and Alberto Garc{\'{\i}}a-Fern{\'{a}}ndez and Aman Anand and Amran Al-Ashouri and Anders Hagfeldt and Andrea Crovetto and Antonio Abate and Antonio Gaetano Ricciardulli and Anuja Vijayan and Ashish Kulkarni and Assaf Y. Anderson and Barbara Primera Darwich and Bowen Yang and Brendan L. Coles and Carlo A. R. Perini and Carolin Rehermann and Daniel Ramirez and David Fairen-Jimenez and Diego Di Girolamo and Donglin Jia and Elena Avila and Emilio J. Juarez-Perez and Fanny Baumann and Florian Mathies and G. S. Anaya Gonz{\'{a}}lez and Gerrit Boschloo and Giuseppe Nasti and Gopinath Paramasivam and Guillermo Mart{\'{\i}}nez-Denegri and Hampus Näsström and Hannes Michaels and Hans Köbler and Hua Wu and Iacopo Benesperi and M. Ibrahim Dar and Ilknur Bayrak Pehlivan and Isaac E. Gould and Jacob N. Vagott and Janardan Dagar and Jeff Kettle and Jie Yang and Jinzhao Li and Joel A. Smith and Jorge Pascual and Jose J. Jer{\'{o}}nimo-Rend{\'{o}}n and Juan Felipe Montoya and Juan-Pablo Correa-Baena and Junming Qiu and Junxin Wang and K{\'{a}}ri Sveinbjörnsson and Katrin Hirselandt and Krishanu Dey and Kyle Frohna and Lena Mathies and Luigi A. Castriotta and Mahmoud. H. Aldamasy and Manuel Vasquez-Montoya and Marco A. Ruiz-Preciado and Marion A. Flatken and Mark V. Khenkin and Max Grischek and Mayank Kedia and Michael Saliba and Miguel Anaya and Misha Veldhoen and Neha Arora and Oleksandra Shargaieva and Oliver Maus and Onkar S. Game and Ori Yudilevich and Paul Fassl and Qisen Zhou and Rafael Betancur and Rahim Munir and Rahul Patidar and Samuel D. Stranks and Shahidul Alam and Shaoni Kar and Thomas Unold and Tobias Abzieher and Tomas Edvinsson and Tudur Wyn David and Ulrich W. Paetzold and Waqas Zia and Weifei Fu and Weiwei Zuo and Vincent R. F. Schröder and Wolfgang Tress and Xiaoliang Zhang and Yu-Hsien Chiang and Zafar Iqbal and Zhiqiang Xie and Eva Unger},
+        title = {An open-access database and analysis tool for perovskite solar cells based on the {FAIR} data principles},
+        journal = {Nat Energy}
+      }
+templates:
+    - The perovskite material with the {reduced_formulas__description} of {reduced_formulas#} has a bandgap of {bandgap#} eV.
```

Comment:
instead of `eV` you could use `{TARGET__units}`

Reply:
but is there currently an advantage in doing this?

Reply:
Not if you won't change this dataset setup anymore or if you won't reuse the templates. ;-)

Reply:
updated.